pkg/network/dns_snooper.go

```diff
@@ -193,7 +192,7 @@ func (s *SocketFilterSnooper) pollPackets() {
 		}
 
 		if err == nil {
-			s.processPacket(data)
+			s.processPacket(data, captureInfo.Timestamp)
```

Comment:
It would be good to have a comment here explaining why we use this timestamp instead of time.Now()

Reply:
@heyronhay , added a comment. Please check if it makes sense.